templates/login.debian9.erb

```diff
@@ -0,0 +1,19 @@
+auth       optional   pam_faildelay.so  delay=3000000
```

Comment:
is this data for a default, vanilla install of debian9?

Reply:
I just checked on another VM, yes it does.